src/test/java/review_tests/automated_tests/TestReviewGateway.java

```diff
@@ -0,0 +1,221 @@
+     */
+    @Test
+    public void testGetReviews(){
+        clearDatabases();
+        try {
+            ArrayList<String> ids = new ArrayList<>();
+            ids.add("1");
+            ids.add("2");
+            ArrayList<Review> actualReviews = reviewGateway.getReviews(ids);
+            assert(reviewsMatch(TEST_REVIEW_1, actualReviews.get(0)) &&
+                    reviewsMatch(TEST_REVIEW_2, actualReviews.get(1)));
+        }catch(IOException e){
+            fail("An IOException caused this test to fail. This should never be reached.");
+        }
+    }
+
+    /*
+    Test that addReview actually adds a review to the database
+     */
+    @Test
+    public void testAddReview(){
+        clearDatabases();
+        String newID = "5";
+        int newStars = 1;
+        String newText = "new text";
+        String newUser = "new username";
+        String newLocation = "new location";
+        Review newReview = new ReviewFactory().create(newID, newStars, newText, newUser, newLocation);
+
+        try{
+            reviewGateway.addReview(newReview);
+            Review returnedReview = reviewGateway.getReview(newID);
+            assert(reviewsMatch(newReview, returnedReview));
+        }catch(IOException e1){
+            fail("An IOException caused this test to fail. This should never be reached.");
+        }catch(ReviewNotFoundException e2){
+            fail("This review was not found in the database. If testGetReview works, addReview is the issue");
+        }
+    }
+
+    /*
+    Test whether updateReview updates the review properly
+     */
+    @Test
+    public void testUpdateReview(){
+        clearDatabases();
+        int newStars = 2;
+        String newText = "new text";
+        TEST_REVIEW_1.setStars(newStars);
+        TEST_REVIEW_1.setText(newText);
+
+        try{
+            reviewGateway.updateReview(TEST_REVIEW_1);
+            Review returnedReview = reviewGateway.getReview(TEST_REVIEW_1.getID());
+            assert(returnedReview.getStars() == newStars && returnedReview.getText().equals(newText));
+        }catch(IOException e1){
+            fail("An IOException caused this test to fail. This should never be reached.");
+        }catch(ReviewNotFoundException e2){
+            fail("This review was not found in the database. If testAddReview passes, this should not be reached");
+        }
+    }
+
+    /*
+    Test whether deleteReview actually removes a review from the database
+     */
+    @Test
+    public void testDeleteReview(){
+        clearDatabases();
+        try{
+            reviewGateway.deleteReview(TEST_REVIEW_2.getID());
+            reviewGateway.getReview(TEST_REVIEW_2.getID());
+            fail("getReview did not throw a ReviewNotFoundException, so the review was not removed from th database");
+        }catch(IOException e1){
+            fail("An IOException caused this test to fail. This should never be reached.");
+        }catch(ReviewNotFoundException e2){
+            return;
```

Comment:
U can just make the ReviewNotFoundException ignored to get rid of the intellij error instead of just returning